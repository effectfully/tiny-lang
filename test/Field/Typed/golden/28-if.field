0 + if T then 0 else 0