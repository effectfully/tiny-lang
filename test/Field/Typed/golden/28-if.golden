SomeOf Field (EAppBinOp Add (EConst 0) (EIf (EConst (UniConst Bool True)) (EConst 0) (EConst 0)))